homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -101,14 +102,23 @@ class SwissPublicTransportSensor(
 
     _attr_attribution = "Data provided by transport.opendata.ch"
     _attr_icon = "mdi:bus"
+    _attr_has_entity_name = True
 
     def __init__(
-        self, coordinator: SwissPublicTransportDataUpdateCoordinator, name: str
+        self,
+        coordinator: SwissPublicTransportDataUpdateCoordinator,
+        unique_config_title: str,
     ) -> None:
         """Initialize the sensor."""
         super().__init__(coordinator)
         self._coordinator = coordinator
-        self._attr_name = name
+        self._attr_name = "Departure"
+        self._attr_unique_id = f"{unique_config_title}_departure"
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, str(self.unique_id))},
```

Comment:
by setting the name to the unique_id of the entity you would be making the unique_id of this device `"something_departure"`. This will make adding more sensors difficult (as you preferably want to use the device to group these devices)
```suggestion
            identifiers={(DOMAIN, unique_id)},
```

Reply:
Yes, of course!